src/Main.java

```diff
@@ -1,5 +1,81 @@
+import javax.sound.midi.Soundbank;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("Задача 1");
+        int clientOs = 0;
+        if (clientOs == 0) {
+            System.out.println("Установите приложение для системы iOS по ссылке");
+        }
+        if (clientOs == 1) {
+            System.out.println("Установите приложение для системы Android по ссылке");
+        }
+        System.out.println("Задача 2");
+        int clientDeviceYear = 2013;
+        if (clientOs == 0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложение для системы iOS по ссылке");
+        } else if (clientOs == 0) {
+            System.out.println("Установите приложение для системы iOS по ссылке");
+        }
+        if (clientOs == 1 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложение для системы Android по ссылке");
+        } else if (clientOs == 1) {
+            System.out.println("Установите приложение для системы Android по ссылке");
+        }
+        System.out.println("Задача 3");
+        int year = 2025;
+        if ((year >= 1584) && ((year % 4 == 0) && (year % 100 != 0) || (year % 400 == 0))) {
```

Comment:
много лишних скобок, которые затрудняют чтение условия
```suggestion
        if (year >= 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```